components/Question.js

```diff
@@ -0,0 +1,68 @@
+import styles from "../styles/components/Question.module.css";
+
+export default function Question({
+  question,
+  index,
+  answers,
+  setAnswers,
+  ownChoice,
+  setOwnChoice,
+}) {
+  function handleChoiceSelection({ target }) {
+    if (target.checked && answers.length >= question.choiceAmountRequired) {
+      return;
+    }
+    setAnswers(target.value);
+  }
+
+  return (
+    <section>
+      <label htmlFor={`Question ${index}`}>{question.question}</label>
+      <ul>
+        {question.choices.map((answer, index) => (
+          <li key={index}>
```

Comment:
Ideally keys should be based on properties that can be tied to the thing you're iterating through, so in this case, I'd opt for `answer`
```suggestion
          <li key={answer}>
```